activemq/publisherAMQP.go

```diff
@@ -0,0 +1,74 @@
+package activemq
+
+import (
+	"context"
+	"github.com/andile-innovation/popcorn/log"
+	"net"
+	"pack.ag/amqp"
+	"time"
+)
+
+type NewAMQPPublisherRequest struct {
+	Host string
+	Port int
+}
+
+func NewAMQPPublisher(request NewAMQPPublisherRequest) AMQPPublisher {
+	return AMQPPublisher{
+
+	}
+}
+
+type AMQPPublisher struct {
+	Client *amqp.Client
+	Conn net.Conn
+}
+
+func (p *AMQPPublisher) Connect(address string) error {
+	client, err := amqp.Dial(address,
+		amqp.ConnSASLPlain("admin", "admin"),
+	)
+	if err != nil {
+		return err
+	}
+
+	p.Client = client
+	return nil
+}
+
+func (p *AMQPPublisher) Publish(destination string, data []byte) error {
```

Comment:
keep arguments the same as the interface:
```suggestion
func (p *AMQPPublisher) Publish(channel string, data []byte) error {
```